sbnobj/Common/Reco/classes_def.xml

```diff
@@ -72,14 +72,24 @@
   <class name="art::Wrapper<art::Assns<float,recob::Shower,void>>" />
   <class name="art::Wrapper<art::Assns<recob::Shower,float,void>>" />
 
-  <class name="sbn::Stub" ClassVersion="11">
+  <class name="sbn::Stub" ClassVersion="16">
+   <version ClassVersion="16" checksum="1338933861"/>
+   <version ClassVersion="15" checksum="961565117"/>
+   <version ClassVersion="14" checksum="2410976659"/>
+   <version ClassVersion="13" checksum="2883383711"/>
+   <version ClassVersion="12" checksum="703271609"/>
```

Comment:
Are the versions 12-16 there due to changes you made during development? If so, probably better to just declare this update version 12 and remove the rest?

Reply:
Fixed.